src/utils/datetime.ts

```diff
@@ -38,3 +38,30 @@ export const getEarliestDate = (dates: string[]): number =>
  * @returns {number}
  */
 export const parseDate = (date: string): number => Date.parse(date);
+
```

Comment:
2 things:

1. This will not be localized since we are hard-coding `en-GB`. To make it localized, you can take a look at `src/utils/locale.ts => toLocalizedDate` (I also suggest we move this method to here since it belongs to `datetime` than to `locale`)

2. I think this deserves a couple of tests inside `src/utils/datetime.test.ts`.